api-admin/src/main/resources/api/openapi.yml

```diff
@@ -1762,6 +1762,24 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/portals/v1/{portalShortcode}/studies/{studyShortcode}/env/{envName}/dataImport/{importId}:
+    get:
```

Comment:
Duplicate Get with additional path parameter.
Wondered if I should make as optional query param and return DataImport list with one DataImport and avoid this duplicate Get API. If no param passed, return All Imports